core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -7,6 +7,7 @@
 package org.opensearch.sql.expression.datetime;
 
 import static java.time.temporal.ChronoField.ALIGNED_WEEK_OF_YEAR;
+import static java.time.temporal.ChronoField.YEAR;
```

Comment:
Could be reverted

Reply:
Removed in https://github.com/opensearch-project/sql/pull/1421/commits/a38212cbef93f40de3e26ff6628fcfdd9b9660c0